pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -376,6 +379,10 @@ stdenv.mkDerivation ({
   preFixup = ''
     # Don't strip $dev/lib/modules/*/vmlinux
     stripDebugList="$(cd $dev && echo lib/modules/*/build/*/)"
+  '' + lib.optionalString (stdenv.hostPlatform.isPower64) ''
+    $STRIP -s $out/zImage || true
+  '' + lib.optionalString (stdenv.hostPlatform.isMips) ''
+    $STRIP -s $out/vmlinux || true
```

Comment:
This is what I was trying to avoid — I'd rather have the stripping done by the kernel build system where it supports it, because they'll do a better job of setting the right flags, I think.

On MIPS, there's the `vmlinux.bin` target, which is stripped.  Does that work?

Reply:
> I'd rather have the stripping done by the kernel build system where it supports it

Okay, if that's your goal I will see what I can do.

> because they'll do a better job of setting the right flags, I think.

Well, I'm not sure I agree with that.  I've been dismayed at how little of the build process machinery is shared across architectures.  In a lot of places they just reimplement the same thing multiple times, and then those implementations drift out of sync with each other.  MIPS is especially bad here.

But I will give it a try.

Reply:
Implemented in latest push.  Currently testing.

Reply:
> On MIPS, there's the `vmlinux.bin` target, which is stripped. Does that work?

No, it doesn't.